cordonbleu-service/src/main/java/com/benromberg/cordonbleu/service/coderepository/GitCommitDetail.java

```diff
@@ -30,7 +30,7 @@
 public class GitCommitDetail {
     private static final int BINARY_FILE_CONTROL_CHARACTER_FACTOR = 100;
     private static final String BINARY_FILE_REPLACEMENT = "Binary file content with MD5 checksum %s";
-    private static final int BINARY_ANALYSIS_LIMIT = 100000;
+    private static final int BINARY_ANALYSIS_LIMIT = 100_000_000;
```

Comment:
Won't merge this change. It would use up to 100 MB of memory just to determine if a file is binary or not.

I know you have trouble with bigger files, but this is not the way to solve the issue. It also doesn't solve the issue for files > 100 MB. Let's solve this in another PR and remove it from the current PR so we can merge and move on.

Reply:
Thanks for your feedback @BenRomberg. I reverted it in b91079fce62e0b9024fbd6a5119ea46a3faf5f13